winlogbeat/tests/system/winlogbeat.py

```diff
@@ -75,8 +76,17 @@ def write_event_log(self, message, eventID=10, sid=None,
         if level is None:
             level = win32evtlog.EVENTLOG_INFORMATION_TYPE
 
-        win32evtlogutil.ReportEvent(source, eventID,
-                                    eventType=level, strings=[message], sid=sid)
+        # Retry on exception for up to 10 sec.
+        t = time.process_time()
```

Comment:
I'm wondering if `time.monotonic()` might lead to more expected behavior.  `time.process_time` doesn't account for sleep, so it might take longer than 10 wall time seconds.  But I'm "OK" with `time.process_time()`

Reply:
Thanks, I pushed that change. Let's see how it tests out.